docker-compose.yml

```
@@ -0,0 +1,46 @@
+version: '3.8'
+
+services:
+  traefik:
+    image: traefik:v3.0
```

To avoid an interactive prompt on some systems, to disambiguate:

```suggestion
    image: docker.io/library/traefik:v3.0
```

Likewise below.